### PR TITLE
Update extra-enforcer-rules to 1.8.0

### PR DESCRIPTION
**Pull request by wendigo**
This brings compatibility with JDK 22-23 class formats.

### CODE REVIEW

**Comment by wendigo** [comment]
cc @electrum @martint

**Review by kokosing — APPROVED** [review]

**Comment by mosabua** [comment]
LGTM

**Comment by wendigo** [comment]
@electrum @martint ptal

**Comment by wendigo** [comment]
ping @electrum

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Airbase 152
   - ASM 9.6 (from 9.5)
 * Plugin updates:
   - git-commit-id-maven-plugin 8.0.0 (from 7.0.0) 
+  - extra-enforcer-rules 1.8.0 (from 1.7.0)
```
Comment (electrum) [inline CHANGES.md:12]: Move to 153

**Comment by wendigo** [comment]
Replaced with https://github.com/airlift/airbase/pull/402

**Pull request closed without merging**